### PR TITLE
Add pytorch example

### PR DESCRIPTION
**Pull request by danielvegamyhre**
Create examples folder and add pytorch example.

### CODE REVIEW

File: config/crd/bases/batch.x-k8s.io_jobsets.yaml [hunk 1]
```diff
@@ -1,9597 +0,0 @@
----
```
Comment (ahg-g) [inline config/crd/bases/batch.x-k8s.io_jobsets.yaml:1]: oh, this file still exists!

File: examples/pytorch.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    network:
+      enableDNSHostnames: true
+    template:
+      spec:
+        parallelism: 4
+        completions: 4
+        backoffLimit: 0
+        template:
+          spec:
+            restartPolicy: Never
```
Comment (ahg-g) [inline examples/pytorch.yaml:17]: remove this just to showcase our defaulting

File: examples/pytorch.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+apiVersion: jobset.x-k8s.io/v1alpha1
+kind: JobSet
+metadata:
+  name: pytorch
+spec:
+  replicatedJobs:
+  - name: workers
+    network:
+      enableDNSHostnames: true
```
Comment (ahg-g) [inline examples/pytorch.yaml:9]: remove this since we default it already

File: examples/pytorch.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+apiVersion: jobset.x-k8s.io/v1alpha1
```
Comment (ahg-g) [inline examples/pytorch.yaml:1]: can we create a pytorch folder, I think other examples may require more than one artifact, so better to put each example in a different folder. 

so something like:
examples/pytorch/resnet.yaml

Comment (danielvegamyhre) [reply examples/pytorch.yaml]: done

File: examples/pytorch.yaml (hunk 3 above)
Comment (danielvegamyhre) [reply examples/pytorch.yaml]: done

File: examples/pytorch/resnet.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+apiVersion: jobset.x-k8s.io/v1alpha1
```
Comment (ahg-g) [inline examples/pytorch/resnet.yaml:1]: revert all files but this one since they are merged in the other pr

Comment (danielvegamyhre) [reply examples/pytorch/resnet.yaml:1]: done

**Pull request merged**